feed.go

```diff
@@ -105,3 +106,19 @@ func (f *Feed) ToRss() (string, error) {
 func (f *Feed) WriteRss(w io.Writer) error {
 	return WriteXML(&Rss{f}, w)
 }
+
+// creates an JSON representation of this feed
+func (f *Feed) ToJSON() (string, error) {
+	j := &JSON{f}
+	return j.ToJSON()
+}
+
+// Writes an JSON representation of this feed to the writer.
```

Comment:
@elithrar I would be happy to go through and update/add some docstrings for the package. I will put those in separate PR in the future.